test/fcntl/test_fcntl_dup.c

```diff
@@ -0,0 +1,29 @@
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <errno.h>
+#include <assert.h>
+
+int
+main() {
+    int fd, dupfd, status, bytes;
```

Comment:
We use llvm coding style which puts the return type on the same line as the function decl, and uses 2-space indention (I know, the codebase is not in a consistent state, but nonetheless).



(you can using `clang-format -i test/fcntl/test_fcntl_dup.c` to have this file automatically formatted if you like)

Reply:
Deleted this file in favor of adding a bit more to `test_fcnt.c`.